### PR TITLE
feature: added product page

### PR DESCRIPTION
**Pull request by poojakelkar**
- Added E-Commerce Product page

Live Link: https://neog-e-comm-website.netlify.app/product/index.html

### CODE REVIEW

File: product/index.html [hunk 1]
```diff
@@ -19,17 +19,18 @@
...
         <link rel="stylesheet" href="productstyles.css" />
         <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="https://neo-ui.netlify.app/styles.css">
         
     </head>
     <body>
-        <nav class="navbar navbar-white bg-white">
-            <p>MyWebsite</p>
+        <nav class="navbar">
+            <p class="head">MyWebsite</p>
```
Comment (SHIKHASINGH1506) [inline product/index.html:27]: header as class name can be more suitable and relevant here

**Review by SHIKHASINGH1506 — APPROVED** [review]
Added suggestion regarding naming convention. Rest are good.

File: product/index.html (hunk 1 above)
Comment (poojakelkar) [reply product/index.html:27]: Ok, Thanks for the suggestion @SHIKHASINGH1506

Comment (HarshnaNHaswani) [reply product/index.html:27]: Can use `<h5>/ <h6>` instead

**Review by HarshnaNHaswani — COMMENTED** [review]
Looks good to me 
Just reduce the box-shadow a bit
use proper tags everywhere

**Comment by poojakelkar** [comment]
ok thanks @HarshnaNHaswani

**Pull request merged**